tests/js/dictionary-tests.js

```diff
@@ -1,810 +0,0 @@
-
-    realm.write(() => {
-      data.x = 3;
-      data.y = 2;
-      data.z = 1;
-    });
-
-    TestCase.assertEqual(typeof data, "object", "Should be an object");
-    TestCase.assertEqual(data.x, 3, "Should be an equals to a.x = 3");
-    TestCase.assertEqual(data.y, 2, "Should be an equals to a.y = 2");
-    TestCase.assertEqual(data.z, 1, "Should be an equals to a.z = 1");
-
-    TestCase.assertEqual(typeof mutable, "object", "Should be an object");
-    TestCase.assertEqual(mutable.x, 3, "Should be an equals to mutable.x = 3");
-    TestCase.assertEqual(mutable.y, 2, "Should be an equals to mutable.y = 2");
-    TestCase.assertEqual(mutable.z, 1, "Should be an equals to mutable.z = 1");
-
-    realm.close();
-  },
-
-  testDictionaryDifferentBackend() {
-    const DictSchemaRef = {
-      name: "Dictionary",
-      properties: {
-        a: "{}",
-        b: "{}",
-      },
-    };
-
-    //Shouldn't throw
-    let realm = new Realm({ schema: [DictSchemaRef] });
-    realm.write(() =>
-      realm.create(DictSchemaRef.name, { a: { x: 1, y: 2, z: 3 }, b: { name: "Caesar", second: "August" } }),
-    );
-
-    let data = realm.objects(DictSchemaRef.name)[0].a;
-    let person = realm.objects(DictSchemaRef.name)[0].b;
-
-    TestCase.assertEqual(typeof data, "object", "Should be an object");
-    TestCase.assertEqual(data.x, 1, "Should be an equals to a.x = 1");
-    TestCase.assertEqual(data.y, 2, "Should be an equals to a.y = 2");
-    TestCase.assertEqual(data.z, 3, "Should be an equals to a.z = 3");
-
-    TestCase.assertEqual(typeof person, "object", "Should also being an object");
-    TestCase.assertEqual(person.name, "Caesar", "Should be an equals to Caesar");
-    TestCase.assertEqual(person.second, "August", "Should be an equals to August");
-
-    realm.close();
-  },
-
-  testDictionary_Javascript_Object_Features() {
-    const DictSchema = {
-      name: "testDictionary_Javascript_Object_Features",
-      properties: {
-        a: "{}",
-      },
-    };
-
-    const dict = { a: { x: 1, y: 2, z: 3 } };
-    let realm = new Realm({ schema: [DictSchema] });
-    realm.write(() => realm.create(DictSchema.name, dict));
-    let point = realm.objects(DictSchema.name)[0].a;
-
-    Object.keys(dict.a).forEach((key) => TestCase.assertEqual(dict.a[key], point[key], `Should be equal: ${key}`));
-    TestCase.assertArraysEqual(Object.values(point).sort(), [1, 2, 3], "Should be an equals to: [1, 2, 3]");
-    TestCase.assertArraysEqual(Object.keys(point).sort(), ["x", "y", "z"], "Should be an equals to: ['x', 'y', 'z']");
-
-    let { x, y, z } = point;
-    TestCase.assertEqual(x, 1, "Should be an equals to: [1,3,2]");
-    TestCase.assertEqual(y, 2, "Should be an equals to: [1,3,2]");
-    TestCase.assertEqual(z, 3, "Should be an equals to: [1,3,2]");
-
-    TestCase.assertEqual(point[Symbol.for("x")], 1, "Should work with symbols");
-
-    realm.close();
-  },
```

Comment:
@JacobOscarGunnarsson it seem to me there's quite a lot of these tests that wasn't migrated?